app/models/pillar.rb

```diff
@@ -2,7 +2,7 @@
 # rubocop:disable Metrics/ClassLength
 class Pillar < ApplicationRecord
   validates :pillar, presence: true
-  validates :value, presence: true
+  validates :value, presence: true, format: { with: /\A[\S].*[\S]\z/ }
```

Comment:
The second `\[S]` forces every entry to have at least 2 characters, which currently breaks on the pillar `lb_mon_retries: "3"` I think.

Reply:
updated, can you test again?